pkg/operator/controller/ingress_status_test.go

```diff
@@ -6,13 +6,200 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-ingress-operator/pkg/manifests"
 
 	operatorv1 "github.com/openshift/api/operator/v1"
 
 	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	toolscache "k8s.io/client-go/tools/cache"
 )
 
+func pendingLBService(owner string) *corev1.Service {
+	return &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "router-" + owner,
```

Comment:
Could use `loadBalancerServiceName`.

(I added this comment in the wrong place earlier.)